tests/js/spec/views/performance/trends.spec.jsx

```diff
@@ -126,6 +127,47 @@ describe('Performance > Trends', function() {
         },
       },
     });
+    trendsMock = MockApiClient.addMockResponse({
+      url: '/organizations/org-slug/events-trends/',
+      body: {
+        meta: {
+          count_range_1: 'integer',
+          count_range_2: 'integer',
+          percentage_count_range_2_count_range_1: 'percentage',
+          percentage_percentile_range_2_percentile_range_1: 'percentage',
+          minus_percentile_range_2_percentile_range_1: 'number',
```

Comment:
Would massaging this data with a django serializer help? It could help give more meaningful names in API responses and allow us to type the front-end better.